src/explore-education-statistics-admin/src/components/editable/__tests__/EditableBlockWrapper.test.tsx

```diff
@@ -0,0 +1,114 @@
+import EditableBlockWrapper from '@admin/components/editable/EditableBlockWrapper';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+
+describe('EditableBlockWrapper', () => {
+  test('renders the child block', () => {
+    render(
+      <EditableBlockWrapper>
+        <div>Child block</div>
+      </EditableBlockWrapper>,
+    );
+
+    expect(screen.getByText('Child block')).toBeInTheDocument();
+  });
+
+  test('renders the edit block button and calls `onEdit` when clicked when `onEdit` is provided', () => {
+    const handleEdit = jest.fn();
+    render(
+      <EditableBlockWrapper onEdit={handleEdit}>
+        <div>Child block</div>
+      </EditableBlockWrapper>,
+    );
+
```

Comment:
Best to assert that the `handleEdit` hasn't been called yet before doing pressing the button to prove that the button triggered it.

Reply:
Done